client.go

```diff
@@ -18,6 +18,14 @@ var (
 	apiUrlV2 = "https://api.aiven.io/v2"
 )
 
+var (
+	// errUnableToCreateAivenClient is a template error for when the client cannot be created.
+	errUnableToCreateAivenClient = "unable to create Aiven client"
+
+	// errTokenIsRequired is the error that is returned when the token is not provided.
+	errTokenIsRequired = fmt.Errorf("%s: AIVEN_TOKEN environment variable is required", errUnableToCreateAivenClient)
```

Comment:
The same here; we only use it in a function that is strictly dev use only. Can we move it to the function? It is not adding any values here, making the function body less readable since I always have to go to this var to see what we are doing.

Reply:
👍